### PR TITLE
Powershell cmdlet updates for Resource Mover GA release

### PR DESCRIPTION
**Pull request by natarajks**
## Description
Cmdlets generated for Resource Mover GA version of swagger

1. Two new cmdlets Invoke-AzResourceMoverBulkRemove, Get-AzResourceMoverRequiredForResources
2. Added alias to Add-AzResourceMoverMoveResource. It is called called Update-AzResourceMoverMoveResource
3. Added directive for not flattening the ResourceSettings object in Add-ResourceMoverMoveResource cmdlet
- no-inline:
    - ResourceSettings
4. Updated documentation
5. Updated tests

## Checklist

- [X ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ X] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [X ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [X ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [X ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/723
          The design review was approved for Public Preview of Resource Mover release. GA release has incremental changes
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
There are no markdown help files

### CODE REVIEW

**Comment by VeryEarly** [comment]
Hi @natarajks,

Please fix failed test case.

File: src/ResourceMover/readme.md [hunk 1]
```diff
@@ -60,15 +60,15 @@ module-name: $(prefix).$(service-name)
 namespace: Microsoft.Azure.PowerShell.Cmdlets.$(service-name)
 clear-output-folder: true
 output-folder: .
-aks: $(repo)/specification/resourcemover/resource-manager/Microsoft.Migrate/preview/2019-10-01-preview
+aks: $(repo)/specification/resourcemover/resource-manager/Microsoft.Migrate/stable/2021-01-01
 input-file:
 	- $(aks)/resourcemovercollection.json
-module-version: 0.1.0
+module-version: 0.8.0
```
Comment (VeryEarly) [inline src/ResourceMover/readme.md:66]: why it's 0.8.0 not 0.2.0?

File: src/ResourceMover/Az.ResourceMover.psd1 [hunk 2]
```diff
@@ -1,7 +1,7 @@
 @{
   GUID = '97d87543-8eef-4574-a70d-7317ec4abe54'
   RootModule = './Az.ResourceMover.psm1'
-  ModuleVersion = '0.1.0'
+  ModuleVersion = '1.0.0'
```
Comment (VeryEarly) [inline src/ResourceMover/Az.ResourceMover.psd1:4]: please also update this

File: src/ResourceMover/readme.md (hunk 1 above)
Comment (natarajks) [reply src/ResourceMover/readme.md]: changed version to 0.1.0 as suggested

**Review by VeryEarly — APPROVED** [review]

**Pull request merged**